hw/xbox/nv2a/nv2a.c

```diff
@@ -162,6 +162,7 @@ const struct NV2ABlockInfo blocktable[] = {
 
 const int blocktable_len = ARRAY_SIZE(blocktable);
 
+// FIXME: Add nv2a_reg_names or remove this code
```

Comment:
I vote to remove this instead.

Adding a list would quickly lead to completionists who leak copyrighted lists.
So despite this being insanely useful, idiots would ruin the fun for others by doing illegal stuff on our behalf.
I'd be fine with this if we used envytool names (but unless the whole codebase uses them, the nvidia names would likely return in PRs).

Reply:
This was not done. A fixup would be nice.
I *really* think we'll run into potential legal issues with this, so I'd rather see it removed before merge.